packages/run-protocol/src/vaultFactory/AttackersGuide.md

```diff
@@ -0,0 +1,75 @@
+# An Attacker's guide to the Vault Factory
+
+This is an incomplete list of potential weak points that an attacker might want to focus
+on when looking for ways to violate the integrity of the Vault Factory. It's here to help
+defenders, as "attacker's mindset" is a good way to focus attention for the defender. The
+list should correspond pretty closely to the set of assurances that the Vault Factory aims
+to support.
+
+The VaultFactory's purpose is to make it possible for users to deposit hard currencies and
```

Comment:
I don't have a firm grasp of what "hard currency" means in this context.

A quick search turns up:

> Hard currency refers to money that is issued by a nation that is seen as politically and economically stable.

That doesn't seem right.